docs/Documentation/Compatibility.md

```diff
@@ -772,10 +773,17 @@ Check whether the player is near a specific MythicMobs entity. The first argumen
 #### SpawnMob: `mspawnmob`
 
 Spawn specified amount of MythicMobs at given location. The first argument is a location defined like `100;200;300;world`. Second is MythicMobs internal name (the one defined in MythicMobs' configuration) followed by a colon and a level. Third one is amount and it's required!
+You can also add the optional argument `target` which will make the mob target the player. 
+
+There is also the `private` argument which will hide the mob from all other players until restart. This does not hide particles or block sound from the mob however.
+
+You can also mark the spawned mob with a keyword using marked: argument. 
+It won't show anywhere, and you can check for only marked mobs in mmobkill objective.
 
 !!! example
     ```YAML
     mspawnmob 100;200;300;world SkeletalKnight:1 5
+    mspawnmob 100;200;300;world SkeletonKing:1 5 target private marked::%player%
```

Comment:
This double ":" is not necessary from my point of view. Therefore I would remove it

```suggestion
    mspawnmob 100;200;300;world SkeletonKing:1 5 target private marked:%player%
```